minio/api.py

```diff
@@ -178,10 +177,9 @@ def __init__(self, endpoint, access_key=None,
         self._credentials = credentials or Credentials(
             provider=Chain(
                 providers=[
-                    Static(access_key, secret_key, session_token),
+                    Static(Value(access_key, secret_key, session_token)),
```

Comment:
Maybe it is enough if we can credentials in this form:

```
  Static(access_key, secret_key, session_token),
```

and create a Value instance inside Static init() function.

Let's review the API with @abperiasamy one last time.

Reply:
I have changed to `Static(access_key, secret_key, session_token=None, expiry=None)`